projects/GEN/GEN_MAIN/src/field/cl_GEN_BSpline_Field.cpp

```diff
@@ -12,7 +12,7 @@
 #include "cl_MTK_Mesh_Pair.hpp"
 #include "cl_MTK_Field.hpp"
 #include "cl_MTK_Field_Discrete.hpp"
-#include "cl_MTK_Mapper.hpp"
+//#include "cl_MTK_Mapper.hpp" GAH break dependency
```

Comment:
I can play around with making the mapper an interface, so hopefully FEM doesn't get brought along and we can include the mapper here just fine. But this will take some work